actors/builtin/miner/policy.go

```diff
@@ -164,7 +163,6 @@ func QualityForWeight(size abi.SectorSize, duration abi.ChainEpoch, dealWeight,
 	sectorSpaceTime := big.Mul(big.NewIntUnsigned(uint64(size)), big.NewInt(int64(duration)))
 	// totalDealSpaceTime = dealWeight + verifiedWeight
 	totalDealSpaceTime := big.Add(dealWeight, verifiedWeight)
-	Assert(sectorSpaceTime.GreaterThanEqual(totalDealSpaceTime))
```

Comment:
If this is an invalid state then perhaps we should throw an error, or validate the inputs before this function is called to be sure the invariant holds.